app/cache/get-cache-key.js

```diff
@@ -0,0 +1,7 @@
+const getCacheKey = (category, value) => {
+  return `${category}:${value}`
```

Comment:
What does this line of code do?

Reply:
In Redis all values are stored under a key.  That string is a key.  In order to introduce some kind of structure to that data, you can use naming conventions for keys which include prefixes.

For example, in this service we have different data results we want to store, we could save our data like this:

`payments:frn:1234567890`
`payments:frn:1234567891`
`holds:frn:1234567890`

Then we can more easily categorise our data.  So basically this is building up a string we can use for that key.

Reply:
Makes sense!